studio/components/RedirectThumbnail.tsx

```diff
@@ -0,0 +1,5 @@
+const RedirectThumbnail = ({ permanent }: { permanent: boolean }) => {
+  return <span style={{ fontSize: "1.5rem" }}>{permanent ? "⮕" : "⇢"}</span>;
```

Comment:
There are some issues with using symbols in text-elements, such as the fact that SRs reads them differently, users might have cognitive challenges, and there's the cultural differences which leads us to understand symbols differently. 

Perhaps we can come up with a better user experience 🤔

Reply:
Omitted permanent redirects, so no need to show this distinction.